cmd/cloud/cluster.go

```diff
@@ -1,17 +1,22 @@
 package main
 
 import (
+	"strings"
+
 	"github.com/spf13/cobra"
 
 	"github.com/mattermost/mattermost-cloud/internal/provisioner"
 )
 
 func init() {
 	clusterCreateCmd.Flags().String("provider", "aws", "Cloud provider hosting the cluster.")
-	clusterCreateCmd.Flags().String("size", "", "The size constant describing the cluster.")
+	clusterCreateCmd.Flags().String("state-store", "dev.cloud.mattermost.com", "The S3 bucket used to store cluster state.")
+	clusterCreateCmd.Flags().String("size", "SizeAlef500", "The size constant describing the cluster.")
+	clusterCreateCmd.Flags().String("zones", "us-east-1a", "The zones where the cluster will be deployed.")
 	clusterCreateCmd.MarkFlagRequired("size")
 
 	clusterDeleteCmd.Flags().String("cluster", "", "The id of the cluster to be deleted.")
+	clusterDeleteCmd.Flags().String("state-store", "dev.cloud.mattermost.com", "The S3 bucket where the cluster state is stored.")
```

Comment:
Worth putting the `--state-store` flag on the `clusterCmd` to share among all cluster commands?

Reply:
I did think about it, but chose not to based on my comment here: https://github.com/mattermost/mattermost-cloud/pull/3#discussion_r275033601

What are your thoughts?